File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/BillableEntityResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class BillableEntityResponse 
{
    public long BillableEntityId { get; set; }
    public string Name { get; set; }
    public ProfileShortResponse Responsible { get; set; }
    public ContactInfoResponse Contact { get; set; }
    public SubscriptionResponse Subscription { get; set; }
    public PaymentTypesResponse Payment { get; set; }
}

File: server-scala-play-server/app/model/BillableEntityResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for BillableEntityResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-11T19:31:43.310890579Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class BillableEntityResponse(
  billableEntityId: Option[Long],
  name: Option[String],
  responsible: Option[ProfileShortResponse],
  contact: Option[ContactInfoResponse],
  subscription: Option[SubscriptionResponse],
  payment: Option[PaymentTypesResponse]
)

object BillableEntityResponse {
  implicit lazy val billableEntityResponseJsonFormat: Format[BillableEntityResponse] = Json.format[BillableEntityResponse]
}

